### PR TITLE
German readme

### PR DESCRIPTION
**Pull request by glumand**
## Proposed changes

First translation of the English README to German.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
None.

## Special notes for your reviewer:
I have not added the links to the German version in the Translations-file, because I did not think I should do so. Also I did not add the new "flags-symbols" because I don't know how this would work.

### CODE REVIEW

File: translations/README-ge.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+[![Twitter Follow](https://img.shields.io/twitter/follow/litmuschaos?style=social)](https://twitter.com/LitmusChaos)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3202/badge)](https://bestpractices.coreinfrastructure.org/projects/3202)
+[![BCH compliance](https://bettercodehub.com/edge/badge/litmuschaos/litmus?branch=master)](https://bettercodehub.com/)
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus?ref=badge_shield)
+[![YouTube Channel](https://img.shields.io/badge/YouTube-Subscribe-red)](https://www.youtube.com/channel/UCa57PMqmz_j0wnteRa9nCaw)
+<br><br><br><br>
+
+#### *Das README in [anderen Sprachen](translations/TRANSLATIONS.md).*
+
+[🇰🇷](translations/README-ko.md) [🇨🇳](translations/README-chn.md)
```
Comment (S-ayanide) [inline translations/README-ge.md:20]: Please add [🇬🇧] (https://github.com/litmuschaos/litmus/blob/master/README.md)
And change the links for the above as such
```suggestion
[🇰🇷](https://github.com/litmuschaos/litmus/blob/master/translations/README-ja.md) [🇨🇳](https://github.com/litmuschaos/litmus/blob/master/translations/README-chn.md)
```

File: translations/README-ge.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+- **Für Entwickler**: Mit Litmus können Chaos-Experimente in der
+  Entwicklungsphase einer Anwendung als Erweiterung zu Modultests
+  oder auch Integrationstests durchgeführt werden.
+- **Für CI-pipelines**: In der kontinuierlichen Integration können
+  Chaos-Experimente als Schritt in der Pipeline genutzt werden um Bugs zu
+  finden.
+  - **Für SREs**: Mit Litmus können Chaos-Experimente geplant durchgeführt
+    werden um möglichst früh Schwächen in Anwendungen zu finden und die
+    Stabilität des Endproduktes zu steigern.
```
Comment (S-ayanide) [inline translations/README-ge.md:58]: This won't be a sub-point, please revert it back as the main point in this list.
```suggestion
- **Für SREs**: Mit Litmus können Chaos-Experimente geplant durchgeführt
  werden um möglichst früh Schwächen in Anwendungen zu finden und die
  Stabilität des Endproduktes zu steigern.
```

**Review by S-ayanide — CHANGES_REQUESTED** [review]
Thanks for sending a PR 🎉 

Can you please make the following changes that have been requested! Thank you.

File: translations/README-ge.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+[![Twitter Follow](https://img.shields.io/twitter/follow/litmuschaos?style=social)](https://twitter.com/LitmusChaos)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3202/badge)](https://bestpractices.coreinfrastructure.org/projects/3202)
+[![BCH compliance](https://bettercodehub.com/edge/badge/litmuschaos/litmus?branch=master)](https://bettercodehub.com/)
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus?ref=badge_shield)
+[![YouTube Channel](https://img.shields.io/badge/YouTube-Subscribe-red)](https://www.youtube.com/channel/UCa57PMqmz_j0wnteRa9nCaw)
+<br><br><br><br>
+
+#### *Das README in [anderen Sprachen](translations/TRANSLATIONS.md).*
+
+[🇰🇷](https://github.com/litmuschaos/litmus/blob/master/translations/README-ko.md) [🇨🇳](https://github.com/litmuschaos/litmus/blob/master/translations/README-chn.md) [gb](https://github.com/litmuschaos/litmus/blob/master/README.md)
```
Comment (S-ayanide) [inline translations/README-ge.md:20]:
```suggestion
[🇰🇷](https://github.com/litmuschaos/litmus/blob/master/translations/README-ko.md) [🇨🇳](https://github.com/litmuschaos/litmus/blob/master/translations/README-chn.md) [🇬🇧](https://github.com/litmuschaos/litmus/blob/master/README.md)
```

**Review by S-ayanide — CHANGES_REQUESTED** [review]
I think you forgot to add a `:` in front of gb. Could you please make that minor modification! Thanks

File: translations/README-ge.md (hunk 3 above)
Comment (glumand) [reply translations/README-ge.md:20]: Jap, sure. I hope it works now. :)

File: translations/README-ge.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+somit SREs Schwachstellen in ihren Entwicklungen zu finden.
+Mit Litmus können Chaos-Experimente in der Staging-Umgebung und gegebenenfalls
+auch in der Produktion durchgeführt werden um Schwachstellen und Bugs zu
+finden.
+
+Litmus nutzt einen cloud-nativen Ansatz um Chaos zu erzeugen, verwalten und zu
+beobachten. Um Chaos zu initialisieren und manipulieren werden die folgenden
+Kubernetes Custom Resource Definitions (kurz CRDs) benötigt:
+- **ChaosEngine**: Mittel, um eine Kubernetes-Anwendung der eine
+  Kubernetes-Node mit dem Chaos-Experiment zu verbinden. Der Chaos-Engine wird
```
Comment (christopherfriedrich) [inline translations/README-ge.md:35]:
```suggestion
  Kubernetes-Node mit dem Chaos-Experiment zu verbinden. Die Chaos-Engine wird
```

File: translations/README-ge.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+  vom Chaos-Operator beobachtet, welcher die Chaos-Experimente ausführt.
+- **ChaosExperiment**: Mittel, um die Konfigurationsparameter eines
+  Chaos-Experiments zu gruppieren. Chaos-Experiment CRs werden vom Operator
+  kreiert wenn Experimente vom Chaos-Enginge ausgeführt werden.
+- **ChaosResult**: Mittel, um die Ergebnisse des Chaos-Experiments zu
+  speichern. Der Chaos-Exporter liest die Ergebnisse aus und exportiert die
+  Messwerte auf einen konfigurierten Prometheus-Server.
+
+Die Chaos-Experimente werden auf <a href="https://hub.litmuschaos.io" target="_blank">hub.litmuschaos.io</a> gehostet. Das ist die zentrale Stelle, an der die Entwicker von Anwendungen oder die Nutzer ihre Chaos-Experimente teilen, sodass diese genutzt werden können um die Belastbarkeit und Sicherheit der Anwendung im Endeinsatz weiter zu verbessern.
```
Comment (christopherfriedrich) [inline translations/README-ge.md:44]:
```suggestion
Die Chaos-Experimente werden auf <a href="https://hub.litmuschaos.io" target="_blank">hub.litmuschaos.io</a> gehostet. Das ist die zentrale Stelle, an der die Entwickler von Anwendungen oder die Nutzer ihre Chaos-Experimente teilen, sodass diese genutzt werden können, um die Belastbarkeit und Sicherheit der Anwendung im Endeinsatz weiter zu verbessern. 
```

File: translations/README-ge.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+![Litmus workflow](/images/litmus-arch_1.png)
+
+## Anwendung
+
+- **Für Entwickler**: Mit Litmus können Chaos-Experimente in der
+  Entwicklungsphase einer Anwendung als Erweiterung zu Modultests
+  oder auch Integrationstests durchgeführt werden.
+- **Für CI-pipelines**: In der kontinuierlichen Integration können
+  Chaos-Experimente als Schritt in der Pipeline genutzt werden um Bugs zu
```
Comment (christopherfriedrich) [inline translations/README-ge.md:54]:
```suggestion
  Chaos-Experimente als Schritt in der Pipeline genutzt werden, um Bugs zu
```

File: translations/README-ge.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+  finden.
+- **Für SREs**: Mit Litmus können Chaos-Experimente geplant durchgeführt
+  werden um möglichst früh Schwächen in Anwendungen zu finden und die
```
Comment (christopherfriedrich) [inline translations/README-ge.md:57]:
```suggestion
  werden, um möglichst früh Schwächen in Anwendungen zu finden und die
```

File: translations/README-ge.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+## Am Chaos Hub mitarbeiten
+
+Erste Informationen findet man in den <a href="https://github.com/litmuschaos/community-charts/blob/master/CONTRIBUTING.md" target="_blank">Guildelines für Mitwirkende des Chaos Hub (Seite aktuell noch auf Englisch)</a>
+
+## Anwender
+
+Informationen befinden sich hier: <a href="https://github.com/litmuschaos/litmus/blob/master/ADOPTERS.md" target="_blank">Anwender von LitmusChaos</a>
+
+(_Senden Sie bitte eine PR an die Seite wenn Sie mit Ihren Anwendungen oder
```
Comment (christopherfriedrich) [inline translations/README-ge.md:74]:
```suggestion
(_Senden Sie bitte eine PR an die Seite, wenn Sie mit Ihren Anwendungen oder
```

File: translations/README-ge.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+Entwicklungen Litmus zum Chaos
+Engineering nutzen_)
+
+## Anmerkungen
+
+Ein paar relevante Dinge, die bei der Arbeit mit Litmus zu beachten sind, sind hier
+gelistet. An den meisten Dingen wird bereits gearbeitet in der
+[ROADMAP](./ROADMAP.nd) kann man dazu mehr finden. Für Details zu den
```
Comment (christopherfriedrich) [inline translations/README-ge.md:82]:
```suggestion
gelistet. Die meisten offenen Punkte sind bereits Teil der [ROADMAP](./ROADMAP.md). Für Details zu den
```

File: translations/README-ge.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+  Kubernetes-Resources auf einem Cluster. Im Falle von air-gapped Umgebungen
+  oder Netzwerken müssen die Chaos-Ressourcen unter dieser Prämisse gehostet
+  werden.
+- Wenn plattform-spezifische Chaos-Experimente (vgl. die der AWS, GCP cloud) durchgeführt
+  werden sollen, werden die nötigen Informationen über Kubernetes Secret
+  bereitgestellt. Andere Optionen Informationen und Daten sicher weiterzugeben
+  sind aktuell noch nicht getestet oder implementiert.
+- Wenn Chaos-Experimente die Docker-API von innerhalb aufrufen, dann muss das
+    Docker-Socket installiert werden. Sinnvolles Management und ein geeigneter
+    Ermessensfreiraum sind empfohlen wenn Entwicker und Admins Zugang zu den
```
Comment (christopherfriedrich) [inline translations/README-ge.md:96]:
```suggestion
    Ermessensfreiraum sind empfohlen wenn Entwicker und Administratoren Zugang zu den
```

File: translations/README-ge.md [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+    Experimenten haben.
+- In (seltenen) Fällen, bei denen die Chaos-Experimente Privileged-Container
+  benutzen werden die empfohlenen Sicherheitsrichtlinien dokumentiert.
```
Comment (christopherfriedrich) [inline translations/README-ge.md:99]:
```suggestion
  benutzen, werden die empfohlenen Sicherheitsrichtlinien dokumentiert.
```

File: translations/README-ge.md [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+
+Litmus nutzt einen cloud-nativen Ansatz um Chaos zu erzeugen, verwalten und zu
+beobachten. Um Chaos zu initialisieren und manipulieren werden die folgenden
+Kubernetes Custom Resource Definitions (kurz CRDs) benötigt:
+- **ChaosEngine**: Mittel, um eine Kubernetes-Anwendung der eine
+  Kubernetes-Node mit dem Chaos-Experiment zu verbinden. Der Chaos-Engine wird
+  vom Chaos-Operator beobachtet, welcher die Chaos-Experimente ausführt.
+- **ChaosExperiment**: Mittel, um die Konfigurationsparameter eines
+  Chaos-Experiments zu gruppieren. Chaos-Experiment CRs werden vom Operator
+  kreiert wenn Experimente vom Chaos-Enginge ausgeführt werden.
```
Comment (christopherfriedrich) [inline translations/README-ge.md:39]:
```suggestion
  kreiert, wenn Experimente vom Chaos-Enginge ausgeführt werden.
```

**Review by christopherfriedrich — CHANGES_REQUESTED** [review]
Great work @glumand . I have noticed some small errors. Could you please fix them?.

File: translations/README-ge.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+somit SREs Schwachstellen in ihren Entwicklungen zu finden.
+Mit Litmus können Chaos-Experimente in der Staging-Umgebung und gegebenenfalls
+auch in der Produktion durchgeführt werden um Schwachstellen und Bugs zu
+finden.
+
+Litmus nutzt einen cloud-nativen Ansatz um Chaos zu erzeugen, verwalten und zu
+beobachten. Um Chaos zu initialisieren und manipulieren werden die folgenden
+Kubernetes Custom Resource Definitions (kurz CRDs) benötigt:
+- **ChaosEngine**: Mittel, um eine Kubernetes-Anwendung der eine
+  Kubernetes-Node mit dem Chaos-Experiment zu verbinden. Das Chaos-Engine wird
```
Comment (christopherfriedrich) [inline translations/README-ge.md:35]:
```suggestion
  Kubernetes-Node mit dem Chaos-Experiment zu verbinden. Die Chaos-Engine wird
```

Comment (glumand) [reply translations/README-ge.md]: Oh sorry, I skipped that one.

File: translations/README-ge.md [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+Ein paar relevante Dinge, die bei der Arbeit mit Litmus zu beachten sind, sind hier
+gelistet. Die meisten offenen Punkte sind bereits Teil der [ROADMAP](./ROADMAP.nd). Für Details zu den
+Einschränkungen bestimmter Experimente empfiehlt sich ein Blick in die
+jeweilige Dokumentation unter [Dokumentation der Experimente](https://docs.litmuschaos.io/docs/pod-delete/).
+
+- Der Chaos-Operator sowie die Chaos-Experimente laufen als
+  Kubernetes-Resources auf einem Cluster. Im Falle von air-gapped Umgebungen
+  oder Netzwerken müssen die Chaos-Ressourcen unter dieser Prämisse gehostet
+  werden.
+- Wenn plattform-spezifische Chaos-Experimente (vgl. die der AWS, GCP cloud) durchgeführt
```
Comment (christopherfriedrich) [inline translations/README-ge.md:89]:
```suggestion
- Wenn plattform-spezifische Chaos-Experimente (vgl. die der AWS, GCP ´Cloud) durchgeführt
```

File: translations/README-ge.md [hunk 15]
```diff
@@ -0,0 +1,136 @@
...
+  werden sollen, werden die nötigen Informationen über Kubernetes Secret
```
Comment (christopherfriedrich) [inline translations/README-ge.md:90]:
```suggestion
  werden sollen, werden die nötigen Informationen über Kubernetes Secrets
```

File: translations/README-ge.md [hunk 16]
```diff
@@ -0,0 +1,136 @@
...
+- Wenn Chaos-Experimente die Docker-API von innerhalb aufrufen, dann muss das
+    Docker-Socket installiert werden. Sinnvolles Management und ein geeigneter
+    Ermessensfreiraum sind empfohlen wenn Entwickler und Administratoren Zugang zu den
+    Experimenten haben.
+- In (seltenen) Fällen, bei denen die Chaos-Experimente Privileged-Container
+  benutzen, werden die empfohlenen Sicherheitsrichtlinien dokumentiert.
+
+## Lizenz
+
+Litmus ist unter Apache License, Version 2.0 zugelassen. Die komplette Lizenz
```
Comment (christopherfriedrich) [inline translations/README-ge.md:102]:
```suggestion
Litmus ist unter der Apache License, Version 2.0 zugelassen. Die komplette Lizenz
```

File: translations/README-ge.md [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+ist auf folgender Seite zu finden: [LIZENZ](./LICENSE). Einige Projekte, die
+von Litmus genutzt werden laufen eventuell unter anderen Lizenzen, bitte
```
Comment (christopherfriedrich) [inline translations/README-ge.md:104]:
```suggestion
von Litmus genutzt werden, laufen eventuell unter anderen Lizenzen, bitte
```

File: translations/README-ge.md [hunk 18]
```diff
@@ -0,0 +1,136 @@
...
+Die Chaos-Experimente werden auf <a href="https://hub.litmuschaos.io" target="_blank">hub.litmuschaos.io</a> gehostet. Das ist die zentrale Stelle, an der die Entwickler von Anwendungen oder die Nutzer ihre Chaos-Experimente teilen, sodass diese genutzt werden können, um die Belastbarkeit und Sicherheit der Anwendung im Endeinsatz weiter zu verbessern. 
+
+![Litmus workflow](/images/litmus-arch_1.png)
+
+## Anwendung
+
+- **Für Entwickler**: Mit Litmus können Chaos-Experimente in der
+  Entwicklungsphase einer Anwendung als Erweiterung zu Modultests
+  oder auch Integrationstests durchgeführt werden.
+- **Für CI-pipelines**: In der kontinuierlichen Integration können
```
Comment (christopherfriedrich) [inline translations/README-ge.md:53]:
```suggestion
- **Für CI-Pipelines**: In der Continuous Integration können
```

File: translations/README-ge.md [hunk 19]
```diff
@@ -0,0 +1,136 @@
...
+
+## Mit Litmus loslegen
+
+[![IMAGE ALT TEXT](images/maxresdefault.jpg)](https://youtu.be/W5hmNbaYPfM)
+
+Erste Informationen findet man in der <a href="https://docs.litmuschaos.io/docs/next/getstarted.html" target="_blank">Litmus Dokumentation (Seite aktuell noch auf Englisch)</a>.
+
+## Am Chaos Hub mitarbeiten
+
+Erste Informationen findet man in den <a href="https://github.com/litmuschaos/community-charts/blob/master/CONTRIBUTING.md" target="_blank">Guildelines für Mitwirkende des Chaos Hub (Seite aktuell noch auf Englisch)</a>
```
Comment (christopherfriedrich) [inline translations/README-ge.md:68]:
```suggestion
Erste Informationen findet man in den <a href="https://github.com/litmuschaos/community-charts/blob/master/CONTRIBUTING.md" target="_blank">Guidelines für Mitwirkende des Chaos Hub (Seite aktuell noch auf Englisch)</a>
```

File: translations/README-ge.md [hunk 20]
```diff
@@ -0,0 +1,136 @@
...
+
+## Anwender
+
+Informationen befinden sich hier: <a href="https://github.com/litmuschaos/litmus/blob/master/ADOPTERS.md" target="_blank">Anwender von LitmusChaos</a>
+
+(_Senden Sie bitte eine PR an die Seite, wenn Sie mit Ihren Anwendungen oder
+Entwicklungen Litmus zum Chaos
```
Comment (christopherfriedrich) [inline translations/README-ge.md:75]:
```suggestion
(_Senden Sie bitte eine PR an die Seite, wenn Sie in Ihren Anwendungen oder während der
Entwicklung Litmus zum Chaos
```

File: translations/README-ge.md [hunk 21]
```diff
@@ -0,0 +1,136 @@
...
+  Kubernetes-Resources auf einem Cluster. Im Falle von air-gapped Umgebungen
+  oder Netzwerken müssen die Chaos-Ressourcen unter dieser Prämisse gehostet
+  werden.
+- Wenn plattform-spezifische Chaos-Experimente (vgl. die der AWS, GCP cloud) durchgeführt
+  werden sollen, werden die nötigen Informationen über Kubernetes Secret
+  bereitgestellt. Andere Optionen Informationen und Daten sicher weiterzugeben
+  sind aktuell noch nicht getestet oder implementiert.
+- Wenn Chaos-Experimente die Docker-API von innerhalb aufrufen, dann muss das
+    Docker-Socket installiert werden. Sinnvolles Management und ein geeigneter
+    Ermessensfreiraum sind empfohlen wenn Entwickler und Administratoren Zugang zu den
```
Comment (christopherfriedrich) [inline translations/README-ge.md:95]:
```suggestion
    Docker-Socket installiert werden. Sinnvolles Management ist empfohlen wenn Entwickler und Administratoren Zugang zu den
```

File: translations/README-ge.md [hunk 22]
```diff
@@ -0,0 +1,136 @@
...
+    Experimenten haben.
+- In (seltenen) Fällen, bei denen die Chaos-Experimente Privileged-Container
+  benutzen, werden die empfohlenen Sicherheitsrichtlinien dokumentiert.
+
+## Lizenz
+
+Litmus ist unter Apache License, Version 2.0 zugelassen. Die komplette Lizenz
+ist auf folgender Seite zu finden: [LIZENZ](./LICENSE). Einige Projekte, die
```
Comment (christopherfriedrich) [inline translations/README-ge.md:103]:
```suggestion
ist auf folgender Seite zu finden: [Lizenz](./LICENSE). Einige Projekte, die
```

File: translations/README-ge.md [hunk 23]
```diff
@@ -0,0 +1,136 @@
...
+von Litmus genutzt werden laufen eventuell unter anderen Lizenzen, bitte
+schauen Sie
+bei der jeweiligen Anwendung nach.
```
Comment (christopherfriedrich) [inline translations/README-ge.md:106]:
```suggestion
bei dem jeweiligen Projekt nach.
```

File: translations/README-ge.md [hunk 24]
```diff
@@ -0,0 +1,136 @@
...
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2Flitmuschaos%2Flitmus?ref=badge_large)
+
+
+Litmus Chaos ist Teil der CNCF Projekte.
+
+[![CNCF](https://github.com/cncf/artwork/blob/master/other/cncf/horizontal/color/cncf-color.png)](https://landscape.cncf.io/selected=litmus)
+
+## Gemeinschaft
+
+Die Litmus Gemeinschaft trifft sich immer am dritten Mittwoch jeden Monats um
```
Comment (christopherfriedrich) [inline translations/README-ge.md:117]:
```suggestion
Die Litmus Community trifft sich immer am dritten Mittwoch jeden Monats um
```

It is already a common word in german.

File: translations/README-ge.md [hunk 25]
```diff
@@ -0,0 +1,136 @@
...
+10:00PM IST oder 9:30 AM PST.
+
+Kommunikationsmedien und Anwendungen, die zum Austausch genutzt werden:
```
Comment (christopherfriedrich) [inline translations/README-ge.md:120]:
```suggestion
Kommunikationsmedien zum Austausch und weitere Informationen:
```

File: translations/README-ge.md [hunk 26]
```diff
@@ -0,0 +1,136 @@
...
+Informationen befinden sich hier: <a href="https://github.com/litmuschaos/litmus/blob/master/ADOPTERS.md" target="_blank">Anwender von LitmusChaos</a>
+
+(_Senden Sie bitte eine PR an die Seite, wenn Sie mit Ihren Anwendungen oder
+Entwicklungen Litmus zum Chaos
+Engineering nutzen_)
+
+## Anmerkungen
+
+Ein paar relevante Dinge, die bei der Arbeit mit Litmus zu beachten sind, sind hier
+gelistet. Die meisten offenen Punkte sind bereits Teil der [ROADMAP](./ROADMAP.nd). Für Details zu den
```
Comment (christopherfriedrich) [inline translations/README-ge.md:81]:
```suggestion
Ein paar relevante Dinge, die bei der Arbeit mit Litmus zu beachten sind, sind im Folgenden
aufgelistet. Die meisten offenen Punkte sind bereits Teil der [Roadmap](./ROADMAP.nd). Für Details zu den
```

File: translations/README-ge.md [hunk 27]
```diff
@@ -0,0 +1,136 @@
...
+Einschränkungen bestimmter Experimente empfiehlt sich ein Blick in die
+jeweilige Dokumentation unter [Dokumentation der Experimente](https://docs.litmuschaos.io/docs/pod-delete/).
```
Comment (christopherfriedrich) [inline translations/README-ge.md:83]:
```suggestion
jeweilige [Dokumentation](https://docs.litmuschaos.io/docs/pod-delete/).
```

File: translations/README-ge.md [hunk 28]
```diff
@@ -0,0 +1,136 @@
...
+
+- Der Chaos-Operator sowie die Chaos-Experimente laufen als
+  Kubernetes-Resources auf einem Cluster. Im Falle von air-gapped Umgebungen
```
Comment (christopherfriedrich) [inline translations/README-ge.md:86]:
```suggestion
  Kubernetes-Resourcen auf einem Cluster. Im Falle von air-gapped Umgebungen
```

**Review by christopherfriedrich — CHANGES_REQUESTED** [review]

**Review by christopherfriedrich — APPROVED** [review]
Looks fine for me as a native speaker. Thanks for your contribution @glumand

cc @S-ayanide

**Comment by glumand** [comment]
Thank you @christopherfriedrich and also thanks for the help. Would you mind marking the request as hacktoberfest-accepted? I will keep on changing requested things nevertheless.

**Review by ksatchit — APPROVED** [review]

**Comment by prithvi1307** [comment]
@glumand Can you please share your email address here for us to send you swags?

**Comment by glumand** [comment]
> @glumand Can you please share your email address here for us to send you swags?

My email address is trundle13@googlemail.com :)

**Pull request merged**